docker/docker-compose.yml

```diff
@@ -81,7 +81,7 @@ services:
       - DATABASE_URL=postgres://user:pass@db:5432/interactive
     # override entrypoint and command
     entrypoint: /app/docker/entrypoints/test.sh
-    command: python -m pytest --cov=. --cov-report html --cov-report term-missing:skip-covered
+    command: bash -c "python -m coverage run --branch --source=interactive,reports,services,tests --module pytest && python -m coverage report || python -m coverage html"
```

Comment:
Would it work (and make sense) to put the `source` args in the `pyproject.toml`?  The `omit` args are already there

Reply:
Oh yes!  And the branch one too.  I copied job-server (which has them there for a Reason™) without thinking, thank you 👍